minimesos/src/test/java/com/containersol/minimesos/DefaultMinimumClusterTest.java

```diff
@@ -38,8 +38,7 @@
 
     @After
     public void after() {
-        DockerContainersUtil util = new DockerContainersUtil(dockerClient);
-        util.getContainers(false).filterByName(HelloWorldContainer.CONTAINER_NAME_PATTERN).kill().remove();
+        cluster.destroy();
```

Comment:
* Are you sure this is enough? Containers, which are started by scheduler, are not part of the cluster. Therefore they are not stopped. If I recall correctly, the scheduler gets killed by `destroy()`, but schedulers remain running
* CLUSTER is annotated by `@ClassRule`, therefore `stop()` will be call through `MesosCluster.after()`